Specifications/catalog/catalog.binding.https.md

```diff
@@ -0,0 +1,134 @@
+# Catalog HTTPS Binding
+
+## 1 Introduction
+
+This specification defines a RESTful API over HTTPS for the [Catalog Protocol].
+
+The OpenAPI definitions for this specification can be accessed [here](TBD).
+
+## 2 Path Bindings
+
+### 2.1 Prerequisites
+
+1. The `<base>` notation indicates the base URL for a catalog service endpoint. For example, if the base catalog URL is `api.example.com`, the URL `https://<base>/catalog/request`
+   will map to `https//api.example.com/catalog/request`.
+
+2. All request and response messages must use the `application/json` media type.
+
+### 2.2 CatalogErrorMessage
+
+In the event of a request error, the catalog service must return an appropriate HTTP code and a [CatalogErrorMessage](./catalog.protocol.md#) in the response body.
+
+| Field   | Type          | Description                                                 |
+|---------|---------------|-------------------------------------------------------------|
+| code    | string        | An optional implementation-specific error code.             |
+| reasons | Array[object] | An optional array of implementation-specific error objects. |
+
+### 2.3 The `catalog/request` endpoint
+
+#### 2.3.1 POST
+
+The [CatalogRequestMessage](catalog.protocol.md#1-catalogrequestmessage) corresponds to `POST https://<base>/catalog/request`:
+
+```
+POST https://provider.com/catalog/request
+
+Authorization: ...
+
+{
+  "@context": {
+    "ids": "https://idsa.org/"
+  },
+  "@type": "ids:CatalogRequest"
+  "ids:filter": {}
+}
+```
+
+The `Authorization` header is optional if the catalog service does not require authorization. If present, the contents of the `Authorization` header are detailed in the
+[Authorization section](#authorization).
+
+The `filter` property is optional. If present, the `filter` property can contain an implementation-specific filter expression or query to be executed as part of the catalog
+request.
+
+#### 2.3.2 OK (200) Response
+
+If the request is successful, the catalog service must return a response body containing a [CatalogMessage](./message/catalog.message.json) which is a profiled DCAT Catalog type
+described by the [Catalog Protocol Specification](catalog.protocol.md).
+
+## 3 Technical Considerations
+
+### 3.1 Authorization
+
+A catalog service may require authorization. If the catalog service requires authorization, requests must include an HTTP `Authorization` header with a token. The contents of  
+the token are undefined by may be an OAUTH2, Web DID, or other access token type.
+
+### 3.2 Versioning
+
+- Versioning will be done via URLs. TBD.
+
+### 3.3 Pagination
```

Comment:
Great to mention this! Let's also take a look together at http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_Preferencemaxpagesizeodatamaxpagesiz and http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_ServerDrivenPaging

Reply:
Sure we can discuss it. OData is an older, complex technology. We may be able to get by with something much simpler and more RESTful.